tests/timings_openmp_03_wires.dat

```diff
@@ -0,0 +1,8 @@
+#num_proc t_adj t_ps
```

Comment:
As before, we don't incorporate benchmark data files into the repository.
Please consider providing it as a gist, or directly on the PR.